common/OpTestSystem.py

```diff
@@ -1008,7 +1008,7 @@ def sys_ipmi_boot_system_to_petitboot(self, i_console):
 
         # Exiting to petitboot shell
         self.console.expect('Petitboot', timeout=BMC_CONST.PETITBOOT_TIMEOUT)
-        time.sleep(20)
+        self.console.expect('x=exit', timeout=10)
```

Comment:
@sammj is that always going to be the case though? Isn't the delay connecting to server really just a product of some things that it synchronously waits on before accepting connections?